### PR TITLE
Lock riakc version, since a strange version at HEAD will break giddyu…

### PR DESCRIPTION
**Pull request by ooshlablu**
…p runs

### CODE REVIEW

**Comment by hmmr** [comment]
+1 2385155

**Comment by hmmr** [comment]
@javajolt I think this is in line with the latest pruning and grafting of branches in riak-erlang-client?

**Comment by javajolt** [comment]
For 2.1 this is probably fine.  For 2.2+, however, we probably need a new tag.

File: rebar.config [hunk 1]
```diff
@@ -15,7 +15,7 @@
         {getopt, ".*", {git, "git://github.com/jcomellas/getopt", {tag, "v0.4"}}},
         {meck, ".*", {git, "git://github.com/basho/meck.git", {tag, "0.8.2"}}},
         {mapred_verify, ".*", {git, "git://github.com/basho/mapred_verify", {branch, "master"}}},
-        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "master"}}},
+        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {tags, "2.1.2"}}},
```
Comment (nickelization) [inline rebar.config]: I think this should be `tag` not `tags`.

**Comment by nickelization** [comment]
Once this issue is fixed, we'll need to merge a similar fix to the develop-2.2 and master branches.

**Comment by ooshlablu** [comment]
@javajolt fixed in another pr

**Pull request closed without merging**